tajo-docs/src/main/sphinx/table_management/compression.rst

```diff
@@ -1,5 +1,23 @@
-*********************************
+***********
 Compression
-*********************************
+***********
 
-.. todo::
\ No newline at end of file
+Using compression can make data size compact, thereby enabling efficient use of network bandwidth and storage. Most of Tajo data formats support data compression feature.
+Currently, compression configuration affects only for stored data format and it is specified when a table is created as table meta information(See `Create Table <../sql_language/ddl.html#create-table>`_).
```

Comment:
I suggest ``it is enabled when a table is created with the proper table property.``.